src/firestore/api.ts

```diff
@@ -329,6 +327,111 @@ export class FirestoreApi {
     logger.info(table.toString());
   }
 
+  /**
```

Comment:
Optional: All of these pretty print methods are a bit out of place in an API file - you could consider moving them to their own file